spec/features/user_has_roles_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+describe "An user" do
+ it "has a registered user role" do
+   user = create(:user)
+   role = Role.create(name: "registered_user")
+   user.roles << role
+   visit login_path
+
```

Comment:
We need to make sure the role of registered user is automatically added when a user is created as well.